vit_jax/configs/models.py

```diff
@@ -99,6 +99,24 @@ def get_s16_config():
   return config
 
 
+@_register
```

Comment:
could you add a new header + "fork" the config from B/16, e.g. adding the following code:

```python

# ViT-X/8
#########


@_register
def get_b8_config():
  """Returns the ViT-B/8 configuration."""
  config = get_b16_config()
  config.name = 'ViT-B_8'
  config.patches.size = (8, 8)
  return config
```